pkg/daemon/daemon.go

```diff
@@ -894,11 +895,6 @@ func (dn *Daemon) pauseMCP() error {
 }
 
 func (dn *Daemon) drainNode() error {
-	if dn.disableDrain {
```

Comment:
@SchSeba Isn't it better to leave the disableDrain code here? It is a bit safer in my opinion since it is within the function that does the draining. (just in case someone forgets that the disableDrain config exists)

Reply:
drainNode will now to return early anymore. The behavior is a bit different.